test/integration/minimal/controls/minimal.rb

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'json'
+
+project_name = attribute('project_name')
+region = attribute('region')
+random_suffix = attribute('random_suffix')
+gcloud_args = "--format json --project #{project_name}"
+_ = `gcloud config set project #{region}`
```

Comment:
What is the purpose of assigning values to `_`?

Reply:
`system()` it is! Here we go:

![the fact that this exists...](https://i.stack.imgur.com/1Vuvp.png)

Reply:
I believe that your original approach would have worked without the assignment, but `#system` works as well.